pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
 homepage = "https://github.com/TRAIS-Lab/dattri"
 
 [project.optional-dependencies]
-fast_jl = ["fast_jl"]
+sjlt = ["sjlt"]
 test = ["build", "pytest", "pre-commit", "ruff", "darglint", "scikit-learn", "pretty_midi", "requests", "matplotlib", "pyparsing", "torch>=2.2.0", "torchvision>=0.17.0"]
```

Comment:
add dependency to sjlt for the test option? In the current version, the tests for sjlt will never be triggered due to the checks of sjlt installation.